plotsScript.py

```diff
@@ -2370,36 +2393,39 @@ def makeRachelPlots(depth='udeep', fontSize=18):
             tick.label.set_fontsize(fontSize)
 
-    dirHome = os.path.expanduser('~')
-    fig.savefig(os.path.join(dirHome, 'Desktop/xdHscComp{0}.png'.format(depth)), dpi=120, bbox_inches='tight')
+    fig.savefig(os.path.join(sgsDir, 'xdHscComp{0}.png'.format(depth)), dpi=120, bbox_inches='tight')
 
 if __name__ == '__main__':
-    #cutsPlots()
-    #colExPlots()
-    #rcPlots(rerun='Cosmos1')
-    #rcPlots(rerun='Cosmos1', snrType='mag', xRange=(23.0, 24.5), asLogX=False, xlim=(18.0, 26.0), xlabel='magnitude', polyOrder=3,
-    #        featuresCuts={1:(None, None)})
-    #rcPlots(rerun='Cosmos1', polyOrder=3, extType='ext', ylim=(-0.02, 0.1), xRange=(25.0, 2000.0), yRange=(-0.1, 0.50),
-    #         ylabel='Mag_psf-Mag_cmodel', featuresCuts={1:(None, 0.1)})
-    #rcPlots(rerun='Cosmos1', polyOrder=3, extType='ext', snrType='mag', ylim=(-0.02, 0.1), xRange=(18.0, 25.2), yRange=(-0.1, 0.50),
-    #        ylabel='mag_psf-mag_cmodel (HSC-I deep)', featuresCuts={1:(None, 0.1)}, asLogX=False, xlim=(18.0, 27.0),
-    #        xlabel='mag_cmodel (HSC-I deep)')
-    #magExtPlots()
-    #extCutRoc()
-    #highPostStarsShape(trainClfs=False)
-    #colExtStarsTom()
-    #plt.show()
-    #hstVsHscSize()
-    #extMomentsCompPlots()
-    #xdFitEllipsePlots()
-    #plotPostMarginals()
-    #xdColorFitScores()
-    #extCorrPlot()
-    #peterPlot()
-    #xdColExtFitScores()
-    #xdColExtSvmScores(trainSvm=True)
-    #makeCosmosWidePlots()
-    #makeCosmosWideScoresPlot()
-    #cosmosWideSvmScores()
-    #makeRachelPlots(depth='udeep')
-    xdColExtFitScoresComb()
+    # cutsPlots()
+    # colExPlots()
+    # rcPlots(rerun='Cosmos1')
+    # rcPlots(rerun='Cosmos1', snrType='mag', xRange=(23.0, 24.5), asLogX=False, xlim=(18.0, 26.0),
+    #          xlabel='magnitude', polyOrder=3, featuresCuts={1:(None, None)})
+    # rcPlots(rerun='Cosmos1', polyOrder=3, extType='ext', ylim=(-0.02, 0.1), xRange=(25.0, 2000.0),
+    #          yRange=(-0.1, 0.50), ylabel='Mag_psf-Mag_cmodel', featuresCuts={1:(None, 0.1)})
+    # rcPlots(rerun='Cosmos1', polyOrder=3, extType='ext', snrType='mag', ylim=(-0.02, 0.1),
+    #         xRange=(18.0, 25.2), yRange=(-0.1, 0.50), ylabel='mag_psf-mag_cmodel (HSC-I deep)',
+    #          featuresCuts={1:(None, 0.1)}, asLogX=False, xlim=(18.0, 27.0),
+    #          xlabel='mag_cmodel (HSC-I deep)')
+    # magExtPlots()
+    # extCutRoc()
+    # highPostStarsShape(trainClfs=False)
+    # colExtStarsTom()
+    # plt.show()
+    # hstVsHscSize()
+    # extMomentsCompPlots()
+    # xdFitEllipsePlots()
+    # plotPostMarginals()
+    # xdColorFitScores()
+    # extCorrPlot()
+    # peterPlot()
+    # xdColExtFitScores(trainClfs=True)
+    # xdColExtSvmScores(trainSvm=True)
+    # makeCosmosWidePlots()
+    # makeCosmosWideScoresPlot()
+    # cosmosWideSvmScores()
+    # makeRachelPlots(depth='udeep')
+    # xdColExtFitScoresComb()
+
+    # xdColExtFitScores()
+    makeRachelPlots(depth='udeep')
```

Comment:
Is that really the appropriate thing to do when executing this file in the general case?